miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSequencerPartitionContainerController.java

```diff
@@ -126,7 +128,7 @@ public ModelAndView setupForm(@PathVariable Long containerId, ModelMap model) th
 
       model.put("formObj", container);
       model.put("container", container);
-      model.put("containerRuns", runService.listByContainerId(container.getId()));
+      model.put("containerRuns", runService.listByContainerId(container.getId()).stream().map(Dtos::asDto).collect(Collectors.toList()));
```

Comment:
Tangent: I was testing this and it looked like there were no runs available ("No matching records found") but it turns out it was just the search bar being populated and I had forgotten/hadn't noticed because I was not in the context of the List Runs page. It's out of scope for this PR, but I'd like to get feedback from the lab about whether they experience these "forgotten filters" as well, and if we should do anything about it. (Unless @djcooke 's suggestion for storing the state of the DataTables will fix this?)
Also a bug with this: deleting the search term removes the filter, but then the search term reappears in the filter bar (but doesn't do the server lookup). 

[JIRA ticket](https://jira.oicr.on.ca/browse/GLT-1937)